features/informatievragen-fields.feature

```diff
@@ -0,0 +1,621 @@
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992077                       |
+      | fields              | indicatieOverleden              |
+      Dan heeft de response een persoon met de volgende 'overlijden' gegevens
+      | naam  | waarde   |
+      | datum | 20151001 |
+
+  Rule: wanneer een veld van verblijfplaats wordt gevraagd, worden de velden geleverd die nodig zijn voor het bepalen van het type verblijfplaats
+    - dit betreft alle velden uit groepen 11 (adres), 12 (locatie) en land, regel 1, regel 2 en regel 3 van groep 13 (verblijf buitenland)
+
+    @gba
+    Abstract Scenario: Gevraagd veld zit in verblijfplaats
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde                |
+      | burgerservicenummer | <burgerservicenummer> |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                                       | waarde                             |
+      | functieAdres.code                          | <functieAdres>                     |
+      | functieAdres.omschrijving                  | <functieAdres omschrijving>        |
+      | datum aanvang adreshouding (10.20)         | <datumAanvangAdreshouding>         |
+      | straatnaam (11.10)                         | <straat>                           |
+      | naam openbare ruimte (11.15)               | <straat>                           |
+      | huisnummer (11.20)                         | <huisnummer>                       |
+      | postcode (11.60)                           | <postcode>                         |
+      | woonplaats (11.70)                         | <woonplaats>                       |
+      | identificatiecode verblijfplaats (11.80)   | <adresseerbaarObjectIdentificatie> |
+      | identificatiecode nummeraanduiding (11.90) | <nummeraanduidingIdentificatie>    |
+      | locatiebeschrijving (12.10)                | <locatiebeschrijving>              |
+      | land.code                                  | <land>                             |
+      | land.omschrijving                          | <land omschrijving>                |
+      | datum aanvang adres buitenland (13.20)     | <datumAanvangAdresBuitenland>      |
+      | regel 1 adres buitenland (13.30)           | <regel1>                           |
+      | regel 2 adres buitenland (13.40)           | <regel2>                           |
+      | regel 3 adres buitenland (13.50)           | <regel3>                           |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                              |
+      | type                | RaadpleegMetBurgerservicenummer     |
+      | burgerservicenummer | <burgerservicenummer>               |
+      | fields              | verblijfplaats.verblijfadres.straat |
+      Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
```

Comment:
Waarschijnlijk hebben we in een ander feature al vermeld dat we geen lege velden leveren maar moeten we dat er hier niet ook nog bij vermelden?

Reply:
kan je uitleggen waarom?

het ging mij erom te tonen dat we alleen van verblijfadres leveren, niet alleen wat expliciet gevraagd is. En dat sommige andere gegevens van de verblijfplaats (zoals functieAdres) niet automatisch worden meegeleverd.
Dat lege velden niet worden geleverd is hier niet relevant. Dat wordt opgelost in de testautomatisering.

Reply:
Ik neem aan dat je bedoeld dat we **alles** van verblijfadres leveren. 
Duidelijk dat het hier niet relevant is. Wellicht sla ik af en toe door in het volledig willen zijn.